modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayBuild.macro

```diff
@@ -11,6 +15,8 @@ definition {
 			locator1 = "TestrayBuild#ROUTINE_ID",
 			value1 = "${routine}");
 
+		TestrayManage.addProductVersionInBuild(key_name = "1.2");
+
```

Comment:
Hi @joelrocha68, I think that you can put a variable on this macro, instead of that value